### DISCUSSION TITLE
Is "on" reserved? "on" field converts to String true in java

### DISCUSSION THREAD

**Discussion by jkoorts** [OAI/OpenAPI-Specification#2993] [Q&A]
The following yaml:

      by:
        type: string
        description: Email Address of User.
      on:
        type: string
        description: Timestamp of creation.
        example: 2018-07-10T11:44:00Z

Using org.openapitools 5.3.0 creates this:

 @JsonProperty("by")
  private String by;

  @JsonProperty("true")
  private String true;

**Comment by webron** [comment]
That's a tooling question, so best asked in the forums of the tool you're using. Most likely they are using a YAML parser that translates `on` to `true` - it should not do that.

**Comment by handrews** [comment]
Closing as tooling issues are out of scope and the spec aspect has been answered.
